script/Ciel/Day 9/09_01_CIEL09_1.txt

```diff
@@ -71,18 +76,24 @@ I should stop thinking about it.
 [sha:1338a89d6fca7bc96ceabdde8ae4325846af33f0]{
 -- Page 5, Offset 22152.
 -- ……きっと今の夢は、深く考えてはいけないものだ。
-...That dream I just had is definitely something that I shouldn't be mulling over deeply.
+That dream I just saw definitely isn't something I should dwell too deeply on...
```

Comment:
not much for this sentence structure in general, so perhaps
> I definitely shouldn't dwell too deeply on that dream.

Reply:
That's cleaner in isolation, but in context- 
> I should stop thinking about it.
> I definitely shouldn't dwell too deeply on that dream.

it feels a bit repetitive. Maybe just pare it down a bit?
> That dream isn't the sort of thing I should dwell too deeply on...